### PR TITLE
Fix remaining tickets maths for giveaway entry email

### PR DESCRIPTION
**Pull request by aliceluo2048**
At the moment, if you have 4 tickets for example, the email message says you need one more ticket for next giveaway entry. 

f(x) = (3 -x ) mod 3

### CODE REVIEW

**Review by justintmckibben — APPROVED** [review]
[quick maths](https://www.youtube.com/watch?v=k3jlviX88iw)

File: app/services/email_manager/giveaway_entries_sender.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def call
         if number_of_tickets % 3 == 0
           entry_status = "You have been entered into this week's giveaway!"
         else
-          entry_status = "You have been entered into this week's giveaway! Only #{number_of_tickets % 3} more tickets until your next entry!"
+          entry_status = "You have been entered into this week's giveaway! Only #{-number_of_tickets % 3} more tickets until your next entry!"
```
Comment (justintmckibben) [inline app/services/email_manager/giveaway_entries_sender.rb:22]: I think you want 3 - number_of_tickets % 3

Comment (Jeremy-Xue) [reply app/services/email_manager/giveaway_entries_sender.rb:22]: omg i am so small brain

**Review by Jeremy-Xue — APPROVED** [review]
thanks for catching this! sorry for the mistake 😭

**Pull request merged**